tools/workspace/x11/repository.bzl

```diff
@@ -15,7 +15,7 @@ def _impl(repo_ctx):
     os_result = determine_os(repo_ctx)
     if os_result.error != None:
         fail(os_result.error)
-    if os_result.is_ubuntu:
+    if os_result.is_ubuntu or os_result.is_manylinux:
```

Comment:
BTW, @jwnimmer-tri, please check this; it's a change I made. I think it's probably right, but just want to make sure you're aware that this was added.